tests/test_metadata.py

```diff
@@ -187,6 +188,7 @@ def _get_container_label_prefix(
         if LOCALHOST.system_info.arch == "x86_64"
         else []
     )
+    + [(cont, "base", ImageType.OS_LTSS) for cont in LTSS_CONTAINERS]
```

Comment:
they're not just called base, there is also base-fips. we need a different solution